src/EffectTransform.js

```diff
@@ -174,6 +178,19 @@ class EffectTransform {
             }
             // gl_FragColor.rgb = convertHSL2RGB(hsl);
             dst.set(hslToRgb(hsl));
+
+            // gl_FragColor.rgb *= gl_FragColor.a + epsilon;
+            dst[0] *= alpha;
```

Comment:
Can you add a comment here saying that we're re-applying pre-multiplied alpha?

Reply:
:+1: